cEP-0010.md

```diff
@@ -0,0 +1,142 @@
+coala's Command Line Interface
+==============================
+
+|Metadata|                                                  |
+|--------|--------------------------------------------------|
+|cEP     |0010                                              |
+|Version |1.0                                               |
+|Title   |coala's Command Line Interface                    |
+|Authors |Alexandru-Sorin Maxim <alexandrus.maxim@gmail.com>|
+|Status  |Proposed                                          |
+|Type    |Feature                                           |
+
+
+Abstract
+--------
+
+This cEP describes the design of coala's Command Line Interface (the action selection screen). It intended to refactor the design released as part of coala 0.8.
+
+Adds functionality alongside with a better interaction with the user. The final goal is for coala CLI to be more friendly, easier to read and use. Adds more functionality depending on what user wants!
+
+Changes
+-------
+
+The following changes shall be made to the CLI:
+
+1. Remove most of the text from the action selection screen
+2. Use letters instead of numbers for selecting actions
+3. Allow chaining actions in a single input
+4. Add a 'selection action for all results' option
+5. Make ApplyPatchAction log the applied patch
+6. Reprompt after running actions only if the action says so
+7. Set 'Do nothing' as the default action
+
+The section below describes why we should make these changes.
+
+Why
+---
+
+### Remove most of the text from the action selection screen
+
+Most of the people who use a CLI program do not read the whole output, only a small part of it. Also, a large output from a command could tire them and could make them stop using our product We want to remove the redundant information, keeping only the necessary information.
+
+### Use letters instead of numbers for selecting actions
+
+Easier to remember an action based on the letters instead of numbers. (Do (n)othing).
+
+### Allow chaining actions in a single input
+
+Users could apply more than one patch in a single input. The idea here is to offer more flexibility to our users. Instead of just 1 number, the user could enter more actions.
+
+### Add a 'select action for all results' option
+
+Now users have the possibility to choose at every result an action. We want to add the possibility to choose an action for all results.
```

Comment:
I find this to be quite confusing. I think "to choose an action for every results" would be a better choice.